go/client/cmd_config_test.go

```diff
@@ -1,354 +0,0 @@
-// Copyright 2015 Keybase, Inc. All rights reserved. Use of
```

Comment:
No.  I think we'd have to rewrite this test, almost entirely.  But I didn't think that was worth it, since the risk of a broken `config set` command is pretty low.